internal/xds/translator/testdata/in/xds-ir/http-route-mirror.yaml

```diff
@@ -13,7 +13,10 @@ http:
       endpoints:
       - host: "1.2.3.4"
         port: 50000
-    mirror:
-      name: "mirror-route-dest"
+    mirrors:
```

Comment:
lets keep this example as is, and create one more test file for multiple-mirrors ?

Reply:
Sound good, will do.